python/raft-dask/raft_dask/include_test/CMakeLists.txt

```diff
@@ -15,6 +15,6 @@
 set(cython_sources raft_include_test.pyx)
 set(linked_libraries raft::raft)
 rapids_cython_create_modules(
-  SOURCE_FILES "${cython_sources}" ASSOCIATED_TARGETS raft LINKED_LIBRARIES "${linked_libraries}"
+  SOURCE_FILES "${cython_sources}" LINKED_LIBRARIES "${linked_libraries}"
                                                                             CXX
```

Comment:
Seems like this doesn't follow CMake style conventions with the `CXX` aligned so far to the right, after removing the `ASSOCIATED_TARGETS raft` piece. Do style checks not complain? Maybe check into that.